filebeat/tests/system/test_crawler.py

```diff
@@ -48,6 +48,9 @@ def test_fetched_lines(self):
         # Check that output file has the same number of lines as the log file
         assert iterations == len(output)
 
+        # Check that the application did not panic
+        assert self.did_not_panic()
+
```

Comment:
I wonder if we can't puț these în the tearDown method to avoid duplication. I guess panics are never good.